README.md

```diff
@@ -19,8 +19,8 @@
 ## Documentation
 
 - [Complete Setup Guide](https://docs.ghost.org/docs/install)
```

Comment:
Does this need to be normalized as well?

Reply:
probably, yeah